src/persistence/job.js

```diff
@@ -0,0 +1,33 @@
+const sql = require('sql-template-strings');
+const {v4: uuidv4} = require('uuid');
+const db = require('./db');
+
+module.exports = {
+  async create(jobParameters) {
+    try {
+      const {rows} = await db.query(sql`
+      INSERT INTO jobs
+        VALUES (${uuidv4()}, ${jobParameters.title}, ${
+        jobParameters.salaryRange
+      }, ${jobParameters.description}, current_date, ${jobParameters.tags}, ${
+        jobParameters.company
+      }, ${jobParameters.logoURL})
+        RETURNING id, title, tags;
+      `);
+      return rows[0];
+    } catch (error) {
+      if (error.constraint === 'users_email_key') {
```

Comment:
Why do we check this?

Reply:
This one is a mistake because I copied it from persistence/users.js and modified without checking.

Reply:
Can you remove it? The constraint `users_email_key` belongs to `users` table which has unique email. For the `jobs` table, we don't have it right?

And please don't copy and paste the code that you don't understand.